test/model-zoo/yolov8n_relu6_age--256x256_quant_tflite_cpu_1/regression.py

```diff
@@ -1,13 +1,14 @@
 import numpy as np
 import json
+from sys import float_info
 
```

Comment:
is this the latest regression postprocessor?
is this updated in the postprocessor repo?